tests/pytest_tests/system_tests/test_artifacts/test_wandb_artifacts.py

```diff
@@ -1325,6 +1325,68 @@ def test_http_storage_handler_uses_etag_for_digest(
         assert entry.ref == "https://example.com/foo.json?bar=abc"
         assert entry.digest == expected_digest
 
+    artifact = wandb.Artifact(type="dataset", name="my-arty")
+    mock_boto(artifact)
+    artifact.add_reference("s3://my-bucket/my_object.pb")
+
+    assert artifact.digest == "8aec0d6978da8c2b0bf5662b3fd043a4"
+    manifest = artifact.manifest.to_manifest_json()
+    assert manifest["contents"]["my_object.pb"] == {
+        "digest": "1234567890abcde",
+        "ref": "s3://my-bucket/my_object.pb",
+        "extra": {"etag": "1234567890abcde", "versionID": "1"},
+        "size": 10,
+    }
```

Comment:
This looks like random copy-paste from `test_add_s3_reference_object` above, remove.

Reply:
I had a bunch of recurring weird merge conflicts from trying to rebase, but even that is a really weird one 🙃 thanks!